display/private.go

```diff
@@ -43,7 +43,9 @@ func followLink(t *tab, prev, next string) {
 
 	if t.hasContent() {
 		t.saveScroll() // Likely called later on, it's here just in case
-		nextURL, err := resolveRelLink(t, prev, next)
+		var nextURL string
+		var err error
+		nextURL, err = resolveRelLink(t, prev, next)
```

Comment:
```suggestion
		nextURL, err := resolveRelLink(t, prev, next)
```

There's no need to add these lines.

Reply:
oops.